bincrafters/build_shared.py

```diff
@@ -112,7 +113,11 @@ def get_conan_upload(username):
 def get_conan_remotes(username):
     remotes = os.getenv("CONAN_REMOTES")
     if remotes:
-        return [remotes.split('@')] if '@' in remotes else [remotes]
+        remotes = remotes.split(',')
+        for remote in reversed(remotes):
+            if '@' in remote:
+                remote = RemotesManager._get_remote_from_str(remote, var_name=remote)
```

Comment:
is result `remote` used anywhere?

Reply:
it's a reference from remotes, it will be changed directly in the list.